asgi_ipc.py

```diff
@@ -42,15 +42,11 @@ def __init__(self, prefix="asgi", expiry=60, group_expiry=86400,
         )
         self.thread_lock = threading.Lock()
         self.prefix = prefix
-        self.channel_store = MemoryDict(
-            "/%s-chan" % self.prefix,
-            size=channel_memory
-        )
+        path = Path('/tmp') / Path('{prefix}.sqlite'.format(prefix=prefix))
+        connection = sqlite3.connect(str(path))
+        self.message_store = MessageTable(connection, '{}-message'.format(prefix))
```

Comment:
Ah, that's a good point about the semaphore path. We want to make sure someone can run two of these on one machine, so there needs to be a prefix in there, it would seem. Running two and testing they don't merge might be a good extra test to add.